docs/getting_started.rst

```diff
@@ -100,25 +169,26 @@ Setup
         }
     }
 
-#. To add recipients for xAPI or Caliper events:
+Router configuration
+--------------------
 
-   #. log in to http://localhost:18000/admin/event_routing_backends/routerconfiguration/add/
+Routers can be configured in django admin settings and require following properties:
```

Comment:
We should mention under which section of django admin these settings can be configured.